assets/js/modules/analytics/common/property-select.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Analytics Property Select component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Select, Option } from '../../../material-components';
+import { STORE_NAME } from '../datastore';
+import { PROPERTY_CREATE } from '../datastore/properties';
+import { isValidAccountID } from '../util';
+
+export default function PropertySelect() {
+	const {
+		accountID: existingTagAccountID,
+		propertyID: existingTagPropertyID,
+	} = useSelect( ( select ) => select( STORE_NAME ).getExistingTag() ) || {};
+	const currentAccountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
+	const accountID = existingTagAccountID || currentAccountID;
+	const currentPropertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+	const propertyID = existingTagPropertyID || currentPropertyID;
```

Comment:
Following up on our call earlier, this would be handled in the datastore, so here we can just rely on `getAccountID` and `getPropertyID`.

The datastore should act as follows:
* If an existing tag, set the account ID and property ID accordingly.
* If a `matchedProperty` is returned from the API, set the account ID and property ID accordingly _if_ neither of them have a value.

Reply:
Yep, I haven't gotten to updating this component yet, but `matchedProperty` should already be handled.